$OpenBSD: patch-extensions_history-list_vala,v 1.1 2015/03/31 06:45:36 jasper Exp $

error: `ListStore' is an ambiguous reference between `GLib.ListStore' and `Gtk.ListStore'

--- extensions/history-list.vala.orig	Tue Mar 31 08:39:44 2015
+++ extensions/history-list.vala	Tue Mar 31 08:39:53 2015
@@ -319,7 +319,7 @@ namespace HistoryList {
         }
 
         private void create_widgets () {
-            ListStore model;
+            Gtk.ListStore model;
             TreeIter iter;
             TreeIter? active_iter = null;
 
@@ -331,7 +331,7 @@ namespace HistoryList {
 
             var tab_closing_behavior = this.hl_manager.get_integer ("TabClosingBehavior");
 
-            model = new ListStore (2, typeof (string), typeof (int));
+            model = new Gtk.ListStore (2, typeof (string), typeof (int));
 
             model.append (out iter);
             model.set (iter, TabClosingBehaviorModel.TEXT, _("Do nothing"),
